src/include/common/config.h

```diff
@@ -27,6 +27,7 @@ extern std::atomic<bool> enable_logging;
 /** If ENABLE_LOGGING is true, the log should be flushed to disk every LOG_TIMEOUT. */
 extern std::chrono::duration<int64_t> log_timeout;
 
+static constexpr const uint64_t INVALID_INDEX = -1;
```

Comment:
-1 for a `uint64_t`, arguably using `std::numeric_limits` is a better idea

Reply:
It's also not used anywhere, I'll check for refsol and remove it. Seems this is added when I attempt to merge a file from DuckDB.